### ISSUE TITLE
PF4 About Modal does not match design

### ISSUE THREAD

**Issue by mcarrano** [patternfly/patternfly-react#723]
I just reviewed the PF4 About modal that was merged today. Unfortunately I did not get to see this before it merged. While the placeholder text fields were added, as requested, these are still not formatting correctly. What I see is this:

![screen shot 2018-10-08 at 2 14 50 pm](https://user-images.githubusercontent.com/6619860/46626351-1442d780-cb05-11e8-9068-4542962637ed.png)

What I should see is this:

![screen shot 2018-10-08 at 2 17 23 pm](https://user-images.githubusercontent.com/6619860/46626366-202e9980-cb05-11e8-9327-41535cfb3f1a.png)

Note that the attributes and values lay out in a horizontal arrangement.

**Comment by mcarrano** [comment]
Hi @tlabaj . Unfortunately I did not get a chance to review this before your PR was merged so I filed this follow-up issue.  Any reason that you were unable to format the body text as in the Core example? See the screen grab above.

**Comment by tlabaj** [comment]
Thanks @mcarrano Matt. That was broken with the live demo merge.  Once the Content component is merged I can update the Modal to us that and fix this issue.

**Comment by mcarrano** [comment]
OK, thanks for explaining @tlabaj . @LHinson FYI

**Comment by tlabaj** [comment]
This depends on issue #692
